sonoff/sonoff.ino

```diff
@@ -2551,4 +2551,5 @@ void loop()
 
 //  yield();     // yield == delay(0), delay contains yield, auto yield in loop
   delay(sleep);  // https://github.com/esp8266/Arduino/issues/2021
+  XsnsCall(FUNC_LOOP);
```

Comment:
Hi, This line shouldn't be there.

Please, check other sensors to see the way tasmota handle this.